modules/lang-painless/src/main/java/org/elasticsearch/painless/Locals.java

```diff
@@ -318,6 +318,19 @@ public Variable(Location location, String name, Type type, int slot, boolean rea
         public int getSlot() {
             return slot;
         }
+
+        @Override
+        public String toString() {
```

Comment:
I added this because I was frustrated that I couldn't see what a `Variable` was while debugging.